src/bin/nydusd/fs_service.rs

```diff
@@ -48,16 +49,10 @@ impl FsBackendCollection {
         // We only wash Rafs backend now.
         let fs_config = match cmd.fs_type {
             FsBackendType::Rafs => {
-                let mut config: serde_json::Value =
-                    serde_json::from_str(&cmd.config).map_err(DaemonError::Serde)?;
-                trim_backend_config!(
-                    config,
-                    "access_key_id",
-                    "access_key_secret",
-                    "auth",
-                    "token"
-                );
-                Some(config)
+                let cfg = ConfigV2::from_str(&cmd.config)
```

Comment:
Could we handle compatibility here?

Reply:
We have handled compabibility issues at ConfigV2::from_str(), which can handle both json/toml/v1/v2 configuration files.